apps/backend/src/constants.ts

```diff
@@ -1 +1,35 @@
+import {
+  LeaderboardEntryType,
+  OAuthMapping
+} from "@types";
+
 export const FRONTEND_URL = process.env.FRONTEND_URL || "http://localhost:3000";
+
+export const OAUTH_APP_MAPPING: Record<string, OAuthMapping> = {
+  "strava": {
+    client_side_fetching: true,
+    token_url: "https://www.strava.com/api/v3/oauth/token",
+    redirect_uri: "http://localhost:3000/oauth/exchange_token&approval_prompt=force&scope=read",
+    id: process.env.NEXT_PUBLIC_OAUTH_STRAVA_CLIENT_ID || "",
+    secret: process.env.NEXT_PUBLIC_OAUTH_STRAVA_CLIENT_SECRET || "",
+    data_options: [
+      {
+        type: LeaderboardEntryType.STRAVA_PREVIOUS_MONTH_RUN_DISTANCE,
+        endpoint: "https://www.strava.com/api/v3/athletes/${user_id}/stats", // TODO: move within function?
+      }
```

Comment:
Data options are used for imports. It's technically should be in the next PR but I figured it was harmless to include here.